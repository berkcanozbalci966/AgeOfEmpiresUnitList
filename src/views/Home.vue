<template>
  <div class="image-container">
    <img :src="require('@/assets/img/ageofempires.jpg')" alt="" />
  </div>
</template>

<script>
export default {
  name: "Home",
};
</script>

<style lang="scss" scoped>
.image-container {
  display: flex;
  justify-content: center;
  img {
    width: 100%;
  }
}
</style>
