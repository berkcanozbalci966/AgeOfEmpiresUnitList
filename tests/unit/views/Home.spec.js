import { mount } from "@vue/test-utils";
import Home from "../../../src/views/Home";

describe("Home view", () => {
  it("Render correctly", () => {
    const component = mount(Home);
    expect(component.html()).toContain("<img");
  });

  it("Render snapshot", () => {
    const component = mount(Home);
    expect(component.html()).toMatchSnapshot();
  });
});
