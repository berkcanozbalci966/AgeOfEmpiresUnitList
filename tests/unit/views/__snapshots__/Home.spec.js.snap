// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home view Render snapshot 1`] = `<div class="image-container"><img src="" alt=""></div>`;
